src/Css/Style.php

```diff
@@ -1440,6 +1546,12 @@ public function set_prop(string $prop, $val, bool $important = false, bool $clea
                 $val = self::$_defaults[$prop];
```

Comment:
`--custom: initial` and `--custom: unset` produce an undefined array-key warning here.